src/components/delete.jsx

```diff
@@ -1,19 +1,26 @@
 import React, { useState } from 'react';
+import DeleteModal from './DeleteModal';
 
-const Delete = () => {
-  const handleDelete = () => {
-    console.log('deleted');
-  };
+const Delete = (props) => {
+  const [open, setOpen] = useState(false);
 
   return (
     <div>
       <button
         className="border-2 border-red-500 text-red-500 p-1 m-1 flex items-center"
-        onClick={() => handleDelete()}
+        onClick={() => setOpen(true)}
+        id="delete-button"
       >
         <p>Delete</p>
         <img src="/icons/icon-delete.svg" />
       </button>
+      <DeleteModal
+        setOpen={setOpen}
+        open={open}
+        id={props.id}
```

Comment:
I think id could be more specific... could this be `commentId`

Reply:
made more specific